dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -62,8 +64,6 @@ derby/10.16.1.1//derby-10.16.1.1.jar
 derbyshared/10.16.1.1//derbyshared-10.16.1.1.jar
 derbytools/10.16.1.1//derbytools-10.16.1.1.jar
 dropwizard-metrics-hadoop-metrics2-reporter/0.1.2//dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
-eclipse-collections-api/11.1.0//eclipse-collections-api-11.1.0.jar
```

Comment:
Were these two dependencies removed in the new version?

Reply:
Yes, it's removed via
- https://github.com/apache/arrow/pull/40904

Reply:
Got